packages/vxrn/src/create.ts

```diff
@@ -518,6 +519,34 @@ export const create = async (optionsIn: VXRNConfig) => {
     defineEventHandler(() => `packager-status:running`)
   )
 
+  router.post(
+    '/symbolicate',
+    defineEventHandler(async (e) => {
+      const body = await readBody(e)
```

Comment:
Doesn't readBody already return json most of the times? 

But if we're sure there are issues in this, we should just readRawBody utf8 and then try parsing it as we do now. So we do not need to do `body as string` anymore.

Reply:
It returns JSON only when `content-type` is `application/json`, unfortunately React Native is sending this as `text/plain`. So I'll go with `readRawBody()` 👍 

Source: https://github.com/unjs/h3/blob/c04c458810e34eb15c1647e1369e7d7ef19f567d/src/utils/body.ts#L157

Reply:
Yea, readRawBody makes more sense.